### ISSUE TITLE
Cannot create toolbox

### ISSUE THREAD

**Issue by fansari** [debarshiray/toolbox#96]
This is was happens when I try to create a toolbox:

[fansari@bat toolbox]$ ./toolbox -v create
toolbox: Fedora generational core is f29
toolbox: base image is fedora-toolbox:29
toolbox: customized user-specific image is fedora-toolbox-fansari:29
toolbox: container is fedora-toolbox-fansari:29
toolbox: checking value /var/run/.heim_org.h5l.kcm-socket (Stream) of property Listen in sssd-kcm.socket
toolbox: parsing value /var/run/.heim_org.h5l.kcm-socket (Stream) of property Listen in sssd-kcm.socket
toolbox: checking if image fedora-toolbox-fansari:29 already exists
Error: error getting image “fedora-toolbox-fansari:29”: unable to find a name and tag match for fedora-toolbox-fansari in repotags: no such image
toolbox: looking for image localhost/fedora-toolbox:29
ERRO[0000] exit status 1
toolbox: looking for image registry.fedoraproject.org/f29/fedora-toolbox:29
toolbox: base image fedora-toolbox:29 resolved to registry.fedoraproject.org/f29/fedora-toolbox:29
toolbox: trying to create working container toolbox-working-container-b178f4f4-524a-11e9-8779-1c1b0d6163d2
toolbox: trying to configure working container toolbox-working-container-b178f4f4-524a-11e9-8779-1c1b0d6163d2
passwd: Note: deleting a password also unlocks the password.
passwd: Note: deleting a password also unlocks the password.
toolbox: trying to create image fedora-toolbox-fansari:29
Getting image source signatures
Copying blob da22bd5bfb28 [======================================] 269.7MiB / 269.7MiB
Copying blob 2b1e21e196ae [======================================] 160.5MiB / 160.5MiB
Copying blob e06601cab6a2 [======================================] 370.0KiB / 370.0KiB
Copying config acfcc0baf5 [======================================] 1.4KiB / 1.4KiB
Writing manifest to image destination
Storing signatures
ERRO[0001] exit status 1
ERRO[0000] exit status 1
toolbox: failed to create image fedora-toolbox-fansari:29

**Comment by tosmi** [comment]
please also provide the fedora version you are using (release / silverblue yes/no) and the podman version (`podman --version`). maybe posting the output of running `bash -x ./toolbox -v create` to a gist would also be helpful.

thanks!
toni

**Comment by fansari** [comment]
I am using Fedora 29 Silverblue.
podman version 1.1.2

[toolbox.log](https://github.com/debarshiray/toolbox/files/3044744/toolbox.log)

**Comment by tosmi** [comment]
hm, toolbox runs `buildah pull registry.fedoraproject.org/f29/fedora-toolbox:29` and then `podman inspect --format '{{index .RepoTags 0}}' --type image fedora-toolbox:29` which fails for reasons unclear to me. i just ran the commands locally on silverblue 30 and it just worked.

i would try to run `podman images` and see if the image `podman inspect` try to inspect is really there. you could also try to `podman rmi` the image and try to run toolbox again.

the question is also which version of toolbox are you using. i'm always running the latest version from git.

**Comment by fansari** [comment]
I have it working now.

**Comment by tosmi** [comment]
cool, could you explain how you fixed it? i think issue #105 might be related...

thanks!

**Comment by fansari** [comment]
Yes - I can explain this. Since the other containers were not so important and I could recreate them I decided to do a fresh start and remove every container stuff out of the way.

This was not so easy: first there was some permission deny message regarding btrfs overlay. I was not able to remove these subvolumes so I did a backup of all data and reformatted everything with xfs so that btrfs overlay issues cannot happen again.

After I was ready I got and error messagt that xfs does not support overlay. So had to remove my ~/.config/container folder and then it worked.

Today I also had to set "setsebool virt_sandbox_use_fusefs on" to be able to start containers. No idea why I did not had this issue before.

Now toolbox is starting and finally I am able to run CodeMeter within it. Now "Fluendo OnePlay DVD" is starting (on the host). But it is hanging and not playing DVDs.

This is the only software I have which is not running on Silverblue. This is perhaps offtopic here but was the reason why I want to test toolbox.
